include/mrdox/Metadata/Function.hpp

```diff
@@ -126,6 +130,15 @@ struct FunctionInfo
         : SymbolInfo(InfoType::IT_function, id_)
     {
     }
+
+private:
+    explicit
+    FunctionInfo(
+        std::unique_ptr<TemplateInfo>&& T)
```

Comment:
`noexcept`?

Reply:
I'm hesitant to add it here because its dependent upon whether any of the default constructors for the non-static data members of `FunctionInfo` throw. Your call